test/kickstart-templates/kickstart-offline.ks.template

```diff
@@ -0,0 +1,108 @@
+lang en_US.UTF-8
+keyboard us
+timezone UTC
+text
+reboot
+
+# Do not enable a network device
+network --no-activate --nodefroute --hostname=REPLACE_HOST_NAME
+
+# Partition disk with a 1GB boot XFS partition and an LVM volume containing a 10GB+ system root
+# The remainder of the volume will be used by the CSI driver for storing data
+#
+# For example, a 20GB disk would be partitioned in the following way:
+#
+# NAME          MAJ:MIN RM SIZE RO TYPE MOUNTPOINT
+# sda             8:0    0  20G  0 disk
+# ├─sda1          8:1    0 200M  0 part /boot/efi
+# ├─sda1          8:1    0 800M  0 part /boot
+# └─sda2          8:2    0  19G  0 part
+#  └─rhel-root  253:0    0  10G  0 lvm  /sysroot
+#
+zerombr
+clearpart --all --initlabel
+part /boot/efi --fstype=efi --size=200
+part /boot --fstype=xfs --asprimary --size=800
+# Uncomment this line to add a SWAP partition of the recommended size
+#part swap --fstype=swap --recommended
+part pv.01 --grow
+volgroup rhel pv.01
+logvol / --vgname=rhel --fstype=xfs --size=REPLACE_LVM_SYSROOT_SIZE --name=root
+
+# Lock root user account
+rootpw --lock
+
+# Configure non-ostree filesystem
+liveimg --url file:///run/install/repo/liveimg.tar.gz
+
+%post --log=/var/log/anaconda/post-install.log --erroronfail
+
+# Set the configuration of MicroShift to include subjectAltNames
+mkdir -p /etc/microshift
+cat - >/etc/microshift/config.yaml <<EOF
+apiServer:
+  subjectAltNames:
+EOF
+
+DEFAULT_ROUTE_IPS="$(ip route show default 2>/dev/null | awk '{print $9}')"
+if [ -n "${DEFAULT_ROUTE_IPS}" ] ; then
+    for r in ${DEFAULT_ROUTE_IPS} ; do
+        echo "    - ${r}" >>/etc/microshift/config.yaml
+    done
+else
+    MAIN_HOST_IP="$(hostname -I | awk '{print $1}')"
+    echo "    - ${MAIN_HOST_IP}" >>/etc/microshift/config.yaml
+fi
+if [ -n "REPLACE_PUBLIC_IP" ]; then
+    echo "    - REPLACE_PUBLIC_IP" >>/etc/microshift/config.yaml
+fi
+
+# The pull secret is mandatory for MicroShift builds on top of OpenShift, but not OKD
+# The /etc/crio/crio.conf.d/microshift.conf references the /etc/crio/openshift-pull-secret file
+mkdir -p /etc/crio
+cat > /etc/crio/openshift-pull-secret <<EOF
+REPLACE_PULL_SECRET
```

Comment:
Offline, the pull secret doesn't have much use right?

Reply:
Removed